paimon-web-ui/src/form-lib/cdc/use-paimon.ts

```diff
@@ -15,109 +15,110 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License. */
 
-import { getAllCatalogs } from "@/api/models/catalog"
-import type { IJsonItem } from "@/components/dynamic-form/types"
-import { useCatalogStore } from "@/store/catalog"
+import { getAllCatalogs } from '@/api/models/catalog'
+import type { IJsonItem } from '@/components/dynamic-form/types'
+import { useCatalogStore } from '@/store/catalog'
 
 export function usePaimon(item: any) {
-	const { t } = useLocaleHooks()
-	const tabType = item.data.tabType
-	let catalogOptions: any = ref([{value:1}])
-	const catalogStore = useCatalogStore()
-	catalogStore.getAllCatalogs().then(()=>{
-		catalogOptions.value = catalogStore.catalogs
-	})
-	const model = reactive({
-		warehouse: '',
-		metastore: '',
-		url: '',
-		other_configs: '',
-		database: '',
-		table_name: '',
-		primary_key: '',
-		partition_column: '',
-		other_configs2: '',
-	})
+  const { t } = useLocaleHooks()
+  const tabType = item.data.tabType
+  let catalogOptions: any = ref([{ value: 1 }])
+  const catalogStore = useCatalogStore()
+  catalogStore.getAllCatalogs().then(() => {
+    catalogOptions.value = catalogStore.catalogs?.map((e) => {
+      return { label: e.label, value: e.key }
+    })
+  })
+  const data = item.data
+  const model = reactive({
+    catalog: data.catalog || '',
+    other_configs: data.other_configs || '',
+    database: data.database || '',
+    table_name: data.table_name || '',
+    primary_key: data.primary_key || '',
+    partition_column: data.partition_column || '',
+    other_configs2: data.other_configs2 || ''
+  })
```

Comment:
You can assign values using extenders, but pay attention to the type
```suggestion
  const model = reactive({ ...data })
```